Client/Assets/CC_Script/FormAutofill.sys.mjs

```diff
@@ -0,0 +1,270 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
I keep seeing this format for these headers. Is Swiftlint not screaming?

Reply:
Not at all, as this is technically not our code. Its added to XCode so we can easily find and make changes IF required.

Reply:
<img width="593" alt="image" src="https://github.com/mozilla-mobile/firefox-ios/assets/8919439/ce3ae7eb-ba81-481b-b1ce-522215c6478e">